lib/rules/sort-requires.js

```diff
@@ -0,0 +1,95 @@
+const buildSortRule = require('../utils/sort-rule-builder');
+
+const isTopLevel = node => node.parent.type === 'Program';
+
+const isStaticRequire = node =>
+  node.type === 'CallExpression' &&
+  node.callee.type === 'Identifier' &&
+  node.callee.name === 'require' &&
+  node.arguments.length === 1;
+
+const isRequire = node => node.type === 'VariableDeclaration' && node.declarations[0].init?.callee?.name === 'require';
+
+const hasVariableDeclarations = node => !!node.declarations;
+
+const hasObjectPattern = node => node.declarations?.[0].id?.type === 'ObjectPattern';
+
+const hasMultipleProperties = node => node.declarations?.[0].id?.properties.length > 1;
```

Comment:
```suggestion
const hasMultipleProperties = node => node.declarations?.[0]?.id?.properties.length > 1;
```

I'm not sure whether this change makes sense though. 😄

Reply:
That’s not needed because declarations is never an empty array.